api/collections.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const QuoteCollection = require('../models/quoteCollection.js');
+const router = require('express').Router();
+const HTTPStatus = require('http-status');
```

Comment:
Pick a naming/import convention and stick with it throughout code base. 
My favourite is all npm/third-party modules first then your own modules separated with blank space. Naming-wise, you can capitalize moudule imports and use lower case when importing single functions. Also, alphabetizing each section improves readability.